client/lib/route/path.ts

```diff
@@ -1,14 +1,15 @@
-/** @format */
 /**
  * External dependencies
  */
-import { includes } from 'lodash';
+import { URL as URLString, SiteSlug, SiteId } from 'types';
 
 /**
  * Internal Dependencies
  */
 import { trailingslashit, untrailingslashit } from './index';
 
+type SiteFragment = SiteSlug | number | false;
```

Comment:
I think we're actually better off not defining this and using `SiteId`, `SiteSlug` and `false` where they're expected.
```suggestion
```